### PR TITLE
[EDU-2401] feat: add migrate to azion PT

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Translated Migrate to Azion 

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-2401]: https://aziontech.atlassian.net/browse/EDU-2401?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+2. **Domínio de entrega**: o domínio alvo da aplicação. Esta saída assume a forma de um domínio Azion, que pode ser apontado para um domínio personalizado. 
+  <Button href="/pt-br/documentacao/produtos/edge-application/domains/" text="Ir para referência de Domains" variant="secondary" target="_blank"></Button>
+
+A maioria das aplicações precisa de uma origem da qual a Azion recupera o conteúdo e o código que as compõem. Essa aplicação passa pelos nós de borda da Azion para ser entregue aos usuários através de um endereço de domínio.
+
+<Button href="/pt-br/documentacao/produtos/guias/build/visao-geral/" text="Ir para a visão geral da edge application" variant="secondary" target="_blank"></Button>
+
+---
+
+## Passo 1: Criar uma edge application
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:33]:
```suggestion
## Passo 1: Crie uma edge application
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Você pode começar criando uma edge application usando um template básico. Uma opção recomendada é o template [Hello World](/pt-br/documentacao/produtos/guides/hello-world-template/), que usa **Edge Functions** para executar uma função simples no edge.
+
+<Button href="/pt-br/documentacao/produtos/marketplace/visao-geral-templates-e-integracoes/" text="Ir para referência de Templates" variant="secondary" target="_blank"></Button>
+
+Veja a [lista completa de templates disponíveis](/pt-br/documentacao/produtos/guias/#azion-templates).
+
+---
+
+## Passo 2: Testar localmente
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:43]:
```suggestion
## Passo 2: Teste localmente
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+---
+
+## Passo 2: Testar localmente 
+
+Antes de implantar a aplicação, a recomendação é [fazer o staging da aplicação através do arquivo hosts](/pt-br/documentacao/produtos/guias/testar-edge-application-atraves-do-arquivo-hosts/) para garantir que tudo está funcionando corretamente. Este passo ajuda a identificar e resolver quaisquer erros de configuração ou problemas causados pela migração.
+
+---
+
+## Passo 3: Apontar o domínio
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:49]:
```suggestion
## Passo 3: Aponte o domínio
```

**Review by hannah-sk — APPROVED** [review]
alguns ajustes e sugestões

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Antes de migrar aplicações para a Azion, existem duas variáveis principais que devem existir:
+
+1. **Endereço de origem**: o servidor de origem da aplicação. Esta entrada para o edge assume a forma de um host, um endereço IPv4 ou IPv6 de um serviço de armazenamento ou uma aplicação existente.
+  <Button href="/pt-br/documentacao/produtos/edge-application/origins/" text="Ir para referência de Origins" variant="secondary" target="_blank"></Button>
+
+2. **Domínio de entrega**: o domínio alvo da aplicação. Esta saída assume a forma de um domínio Azion, que pode ser apontado para um domínio personalizado. 
+  <Button href="/pt-br/documentacao/produtos/edge-application/domains/" text="Ir para referência de Domains" variant="secondary" target="_blank"></Button>
+
+A maioria das aplicações precisa de uma origem da qual a Azion recupera o conteúdo e o código que as compõem. Essa aplicação passa pelos edge nodes da Azion para ser entregue aos usuários através de um endereço de domínio.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:27]:
```suggestion
A maioria das aplicações precisa de uma origem da qual a Azion recupera o conteúdo e o código que as compõem. Essa aplicação passa pelos edge nodes da Azion para ser entregue aos usuários através de um domínio.
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+<Button href="/pt-br/documentacao/produtos/guias/build/visao-geral/" text="Ir para a visão geral da edge application" variant="secondary" target="_blank"></Button>
+
+---
+
+## Passo 1: Crie uma edge application
+
+Você pode começar criando uma edge application usando um template básico. Uma opção recomendada é o template [Hello World](/pt-br/documentacao/produtos/guides/hello-world-template/), que usa **Edge Functions** para executar uma função simples no edge.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:35]:
```suggestion
Você pode começar criando uma edge application usando um template básico e o endereço de origem de sua aplicação. Uma opção recomendada é o template [Hello World](/pt-br/documentacao/produtos/guides/hello-world-template/), que usa **Edge Functions** para executar uma função simples no edge.
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+description: Aprenda como você pode trazer suas aplicações e infraestrutura para a Azion.
+meta_tags: azion, migrar, domains, staging, domínio
+namespace: documentation_migrate_to_azion
+permalink: /documentacao/produtos/migrar-para-a-azion/
+---
+
+Este guia fornece um passo a passo para migrar aplicações existentes para a Azion. Ele descreve os requisitos necessários e detalha os passos para uma transferência bem-sucedida de aplicações de outros serviços para o edge.
+
+:::note
+Se sua conta está sob um plano de serviço **Business** ou superior, você tem acesso a especialistas por meio dos [Integration Services](/pt-br/documentacao/servicos/solutions-lab/integration-services/). Para mais informações detalhadas sobre os planos, veja os [Planos de Serviço da Azion](https://www.azion.com/pt-br/precos/#planos).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:12]:
```suggestion
Se sua conta está sob um plano de serviço **Business** ou superior, você tem acesso a especialistas por meio dos [Integration Services](/pt-br/documentacao/servicos/solutions-lab/integration-services/). Para mais informações detalhadas, veja os [Planos de Serviço da Azion](https://www.azion.com/pt-br/precos/#planos).
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+:::
+
+---
+
+## Antes de começar
+
+Antes de migrar aplicações para a Azion, existem duas variáveis principais que devem existir:
+
+1. **Endereço de origem**: o servidor de origem da aplicação. Esta entrada para o edge assume a forma de um host, um endereço IPv4 ou IPv6 de um serviço de armazenamento ou uma aplicação existente.
+  <Button href="/pt-br/documentacao/produtos/edge-application/origins/" text="Ir para referência de Origins" variant="secondary" target="_blank"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:22]:
```suggestion
  <Button href="/pt-br/documentacao/produtos/edge-application/origins/" text="saiba mais sobre Origins" variant="secondary" target="_blank"></Button>
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+2. **Domínio de entrega**: o domínio alvo da aplicação. Esta saída assume a forma de um domínio Azion, que pode ser apontado para um domínio personalizado. 
+  <Button href="/pt-br/documentacao/produtos/edge-application/domains/" text="Ir para referência de Domains" variant="secondary" target="_blank"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:25]:
```suggestion
  <Button href="/pt-br/documentacao/produtos/edge-application/domains/" text="saiba mais sobre Domains" variant="secondary" target="_blank"></Button>
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+A maioria das aplicações precisa de uma origem da qual a Azion recupera o conteúdo e o código que as compõem. Essa aplicação passa pelos edge nodes da Azion para ser entregue aos usuários através de um domínio.
+
+<Button href="/pt-br/documentacao/produtos/guias/build/visao-geral/" text="Ir para a visão geral da edge application" variant="secondary" target="_blank"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:29]:
```suggestion
<Button href="/pt-br/documentacao/produtos/guias/build/visao-geral/" text="saiba mais sobre edge applications" variant="secondary" target="_blank"></Button>
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+---
+
+## Passo 1: Crie uma edge application
+
+Você pode começar criando uma edge application usando um template básico e o endereço de origem de sua aplicação. Uma opção recomendada é o template [Hello World](/pt-br/documentacao/produtos/guides/hello-world-template/), que usa **Edge Functions** para executar uma função simples no edge.
+
+<Button href="/pt-br/documentacao/produtos/marketplace/visao-geral-templates-e-integracoes/" text="Ir para referência de Templates" variant="secondary" target="_blank"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:37]:
```suggestion
<Button href="/pt-br/documentacao/produtos/marketplace/visao-geral-templates-e-integracoes/" text="saiba mais sobre Azion Templates" variant="secondary" target="_blank"></Button>
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+Antes de implantar a aplicação, a recomendação é [fazer o staging da aplicação através do arquivo hosts](/pt-br/documentacao/produtos/guias/testar-edge-application-atraves-do-arquivo-hosts/) para garantir que tudo está funcionando corretamente. Este passo ajuda a identificar e resolver quaisquer erros de configuração ou problemas causados pela migração.
+
+---
+
+## Passo 3: Aponte o domínio
+
+O passo final para migrar totalmente sua aplicação para a Azion é apontar seu domínio personalizado para a edge application da Azion. Ao fazer isso, você permite que os usuários acessem sua edge application através de seu domínio, reduzindo a latência e os custos de processamento na origem.
+
+<Button href="/pt-br/documentacao/produtos/guias/apontar-dominio-para-a-azion/" text="Como apontar um domínio para a Azion"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/migrar-para-a-azion/migrar-para-a-azion.mdx:53]:
```suggestion
<Button href="/pt-br/documentacao/produtos/guias/apontar-dominio-para-a-azion/" text="consulte o guia de como apontar um domínio"></Button>
```

**Review by MarianaAguilera — APPROVED** [review]
sugestões para os botões.

**Pull request merged**